metrics.go

```diff
@@ -481,6 +481,16 @@ type BatchJobMetrics struct {
 
 	// Jobs by ID.
 	Jobs map[string]JobMetric
+
+	// Job Status by Type
+	Status map[BatchJobType]BatchJobSummaryStatus
```

Comment:
Can we not add this in `JobMetric` ? Because the `JobMetric.InProgress` is where I would expect this.

Also not sure why these are `float64`

Reply:
I have nested `Status` within `JobMetric`. Also removed these `float64`s.